[BUG] none repo for qbittorrent

### What happened?

Steps to reproduce:

1. Install package 'qbittorrent'
2. Go to 'none repo version'
3. Error ''ERROR     The cmake build of qbittorrent did not complete successfully"

Expected results: qbittorrent to be installed

### Swizzin commit

0acf022

### What OS are you using?

Ubuntu 22.04 (Jammy)

### What architecture is your OS?

amd64

### Relevant logs and output

```plaintext
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt6LinguistTools/Qt6LinguistToolsTargets.cmake:131 (message):
  The imported target "Qt6::lconvert" references the file

     "/usr/lib/qt6/bin/lconvert"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/lib/x86_64-linux-gnu/cmake/Qt6LinguistTools/Qt6LinguistToolsTargets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Qt6LinguistTools/Qt6LinguistToolsConfig.cmake:47 (include)
  /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake:219 (find_package)
  src/CMakeLists.txt:47 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/qbittorrent/build/CMakeFiles/CMakeOutput.log".
[Sun Jul 23 11:57:40] ERROR     The cmake build of qbittorrent did not complete successfully
[Sun Jul 23 11:57:40]   Please consult the above and/or check the log (less -R +G /root/logs/swizzin.log)
```

--- a/sources/functions/qbittorrent
+++ b/sources/functions/qbittorrent
@@ -203,7 +203,7 @@ function install_qt() {
     if [[ -z ${qbt_use_qt6} ]]; then
         LIST='qtbase5-dev qttools5-dev'
     else
-        LIST='qt6-base-dev qt6-tools-dev'
+        LIST='qt6-base-dev qt6-tools-dev qt6-l10n-tools qt6-tools-dev-tools'
     fi
     qt_repo_version=$(get_candidate_version ${LIST[0]})
     if ! dpkg --compare-versions ${qt_repo_version} ge ${qt_minimum_version}; then

